docs/posts/announcing-powershell-generator.md

```diff
@@ -0,0 +1,84 @@
+
+### Known Issues
+As with all `beta` software, there are bound to be a few glitches or things that are not working. 
+
+We've cataloged some [known issues](https://github.com/Azure/autorest/blob/master/docs/powershell/release-notes.md#caveats-and-known-issues) with this first beta we encourage you to read before reporting any issues you experience.
+
+### Support 
+We're working as fast as we can to finish up the generator, as we have a lot of modules to generate internally. 
+
+If you run into problems, you can post an issue on the [github repo](https://github.com/Azure/autorest/issues) and tag it with the `powershell` label, and we'll try to take a look.
+
+## Features
+
+### Modules work on both Windows PowerShell and PowerShell 
+Due to the use of `netstandard2.0` and `PowerShellStandard.Library`, once compiled, the cmdlets work on both Windows PowerShell 5.1 and PowerShell 6.x.
+
+### Generates modules from OpenAPI files without any external dependencies
+Most language SDKs generated with AutoRest required the use of at least a 'client runtime' package, and often pulls in a few other libraries (ie, `JSON.NET`) that are required to compile the output of the generator.
+
+The new PowerShell generator creates modules that require _no dependencies_ outside of `netstandard2.0` and the `PowerShellStandard.Library` which drastically reduces the chances of having assembly loading conflicts.
+
+### Cmdlets have no weird base-classes or force hierarchy
+All the generated cmdlets inherit `PSCmdlet` and are fairly straightforward. For ARM resources, we already support generating `-AsJob` support for long-running-operations, and this can be expanded in the future to support more patterns.
+
+### An incredible number of extensibility points 
+After generation of a module, the developer may wish to augment the module in many ways (custom work when the module loads, changing the HTTP pipeline, adding additional variants of cmdlets, and more). 
+The generated cmdlets offer number of ways to be customized and enhanced, and we'll be posting some documentation on how to do that in the near future.
+
+### Many variants of cmdlets are created to offer several `ParameterSets`
+Behind-the-scenes, many different flavors of a cmdlet can get created, and these are tied together into a single cmdlet with multiple parameter sets. These can be joined with manually written cmdlets that are written in `.ps1` scripts or C# classes.
+
+### No reflection for serialization
+The generated module has custom-created JSON serialization (using an embedded copy of [Carbon.JSON](https://github.com/carbon/Data/tree/master/Carbon.Json) This significantly improves serialization performance.
+
+## FAQs
+
+### What happened to 'PSSwagger'?
+In order to get to the point where we can generate the [Az](https://azure.microsoft.com/en-us/blog/azure-powershell-az-module-version-1/) 
```

Comment:
Current text doesn't answer the question directly.
We should be comfortable in calling that all future investments will be in PowerShell from Autorest and no new work will be done in PSSwagger codebase. @edyoung your suggestions/feedback here will be helpful.